code/game/machinery/Sleeper.dm

```diff
@@ -209,9 +209,9 @@
 	dir = 8
 	var/orient = "LEFT" // "RIGHT" changes the dir suffix to "-r"
 	var/mob/living/carbon/human/occupant = null
-	var/possible_chems = list(list("inaprovaline", "stoxin", "dexalin", "bicaridine", "kelotane"),
-								   list("inaprovaline", "stoxin", "dexalinp", "bicaridine", "dermaline", "kelotane", "imidazoline"),
-								   list("inaprovaline", "stoxin", "dexalinp", "bicaridine", "dermaline", "kelotane", "imidazoline", "anti_toxin", "ryetalyn", "alkysine", "arithrazine"))
+	var/possible_chems = list(list("epinephrine", "morphine", "salbutamol", "styptic_powder"),

```

Comment:
knew I forgot to test something; need to check of this procs INGEST when it adds to the person or not--if it does, yeah, problem. If it doesn't, then it should be fine.